TYPO3.Neos/Classes/TYPO3/Neos/Eel/FlowQueryOperations/SortOperation.php

```diff
@@ -0,0 +1,117 @@
+<?php
+namespace TYPO3\Neos\Eel\FlowQueryOperations;
+
+/*
+ * This file is part of the TYPO3.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Eel\FlowQuery\FlowQuery;
+use TYPO3\Eel\FlowQuery\Operations\AbstractOperation;
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\TYPO3CR\Domain\Model\Node;
+use TYPO3\TYPO3CR\Domain\Model\NodeInterface;
+
+/**
+ * "sort" operation working on TYPO3CR nodes.
+ * Sorts nodes by specified node properties.
+ */
+class SortOperation extends AbstractOperation
+{
+
+    /**
+     * {@inheritdoc}
+     *
+     * @var string
+     */
+    protected static $shortName = 'sort';
+
+    /**
+     * {@inheritdoc}
+     *
+     * @var integer
+     */
+    protected static $priority = 1;
```

Comment:
Are you sure about that @aertmann ? The docs say "The priority of operations. higher numbers override lower ones.". In fact this can be removed since the default in AbstractOperation is 1 as well.